docs/development.md

```diff
@@ -67,6 +67,16 @@ By default we download the latest version of libtorch. We have an experimental p
   -DLIBTORCH_VARIANT=shared # Set the variant of libtorch to build / link against. (`shared`|`static` and optionally `cxxabi11`)
 ```
 
+#### Windows
+
+On Windows, compile Torch-MLIR with the Microsoft Visual Studio toolchain by
+using the "Developer Command Prompt for Visual Studio".  Here is a sample
+invocation of cmake for Torch-MLIR on Windows.
+
+```shell
+cmake -GNinja -Bbuild -DCMAKE_BUILD_TYPE=Release -DPython3_FIND_VIRTUALENV=ONLY -DLLVM_ENABLE_PROJECTS=mlir -DLLVM_EXTERNAL_PROJECTS="torch-mlir;torch-mlir-dialects" -DLLVM_EXTERNAL_TORCH_MLIR_SOURCE_DIR=%cd% -DLLVM_EXTERNAL_TORCH_MLIR_DIALECTS_SOURCE_DIR=%cd%\externals\llvm-external-projects\torch-mlir-dialects -DMLIR_ENABLE_BINDINGS_PYTHON=ON -DLLVM_TARGETS_TO_BUILD=host externals\llvm-project\llvm
```

Comment:
just curious if we can use `-DLLVM_EXTERNAL_TORCH_MLIR_SOURCE_DIR=. -DLLVM_EXTERNAL_TORCH_MLIR_DIALECTS_SOURCE_DIR=.\externals\llvm-external-projects\torch-mlir-dialects`